.github/ISSUE_TEMPLATE/qg-checklist.md

```diff
@@ -8,26 +8,27 @@ assignees: ''
 ---
 
 ## QG checks
-Please keep this issue open until QG is concluded and will be managed by the Issue Creator!
-We will inform you about finding and proposals in separated issues, this issue here is for the Overview of the Checks!
 
-### Please keep this issue open until QG is concluded!
+Please open and fill in this issue in your product repository to document the compliance with our Tractus-X Release Guideline (TRGs)
 
-Product Owner: <!-- Note: Please search for the Product Owner of this product. -->  
-Dev SPOC: <!-- Note: Please add every contributor to this product -->  
-Helm Chart Version: <!-- Note: Please note the current Helm Chart Version to check. -->  
-App Version: <!-- Note: Please note the current App Version to check. -->  
+**Show compliance with TRGs** by referencing to a tagged link in the respective repository where possible, example: [github.com/eclipse-tractusx/example-repo/tree/1.0.0/docs](https://github.com/eclipse-tractusx/example-repo/tree/1.0.0/docs)
```

Comment:
While quick reading I've been confused. Would coin it a little more
```suggestion
**Show compliance with TRGs** by referencing to a tagged link in the respective repository where possible, example: TRG 1.01 (see [github.com/eclipse-tractusx/example-repo/tree/1.0.0/README.md](https://github.com/eclipse-tractusx/example-repo/tree/1.0.0/README.md))
```

Reply:
thank you, done